src/models/User.js

```diff
@@ -2,16 +2,13 @@ module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define(
     'User',
     {
```

Comment:
I would like to know why you remove the user "id" field ?

Reply:
You are not passing the Id from your form, sequelize already handles that, so it not needed here.